### ISSUE TITLE
[Request] Possible App for Odysse

### ISSUE THREAD

**Issue by clrizzi** [yuliskov/SmartTubeNext#2708]
Hello, Mr Yuliskov! 

I love Smart Tube Next. Forgive me if this is not the ideal place for this, but I would like to ask: Is there any chance to also develop an Android TV app for Odysee?

I found [this project](https://github.com/linimin/lbry-androidtv) but It's not working properly.

I would be happy if you could consider this option.

Thank you!

Greetings.

**Comment by DanTheOracle** [comment]
this would be wonderful.

**Comment by stokito** [comment]
The Odysee has an official app https://play.google.com/store/apps/details?id=com.odysee.app

**Comment by DanTheOracle** [comment]
> The Odysee has an official app https://play.google.com/store/apps/details?id=com.odysee.app

dosnt work on tv's and smarttube only works on tv's as far as i understand it

**Comment by stokito** [comment]
on tv I used Kodi.tv with lbry plugin https://github.com/accumulator/plugin.video.lbry
but it needs for manual configuration of api url

**Comment by DanTheOracle** [comment]
> on tv I used Kodi.tv 

isnt kodi its own operating system [based on the old xbox media player?]

or if it something you somehow run on android os?

**Comment by DanTheOracle** [comment]
> but it needs for manual configuration of api url
 
ok i tried the one it seemed to say in the documentation https://api.lbry.tv/api/v1/proxy and it dosnt work, do you happen to know a step by step or have better info? i just get an api error.

**Comment by clrizzi** [comment]
> > The Odysee has an official app https://play.google.com/store/apps/details?id=com.odysee.app
> 
> dosnt work on tv's and smarttube only works on tv's as far as i understand it

Yeah... I tried using the native mobile app on the TV but it didn't work very well.
